example/client.js

```diff
@@ -1,11 +1,17 @@
 // @flow
 import createClient from '../src';
 
-
 const client = createClient('SimpleStepPublisher', 'A simple chorus step publisher');
 export default client;
 
-export const clientOpened: Promise<void> =
+// Detect whether chorus was enabled by setting a http param in the URL when loading the app
```

Comment:
should that read as *by getting a query param* rather than *by setting a http param*

Reply:
That does read better, changed it.